### DISCUSSION TITLE
GitHub Student Pack Qyery

### DISCUSSION THREAD

**Discussion by techy4shri** [community/community#138695] [GitHub Education]
### Hi everyone,

My institute doesn't provide an .edu domain email which I see is necessary for student package. Undergraduate students don't get it. I have my ID card, update and showing that it is valid but the page asks for specific email. How do I proceed?

**Comment by AryanK1511** [comment]
You should be able to log in with your school credential so for instance I attend seneca and I logged in using myemail@myseneca.ca. If you do this, you should be able to get GitHub Pro

**Comment by Chanduporalla** [comment]
Hi everyone, iam poralla chandu

If your institute doesn't provide a .edu email address, you can still apply for the GitHub Student Developer Pack using an alternative method.

    Use a school-issued document: GitHub allows you to upload proof of your student status. Since you have a valid student ID card, you can submit a clear image of your ID during the verification process. Make sure the ID shows your name, the name of your institution, and your enrollment status.

    Apply with your current email: You can use any email address (even if it's not a .edu email). When prompted for a .edu domain, select the option that says, “I don’t have a school-issued email” or similar, and GitHub will allow you to upload your student ID or another form of verification.

    Other documents: If needed, you can also use enrollment letters, transcripts, or other documents provided by your school that prove your current student status.

This should allow you to proceed with your application. The GitHub team typically reviews and approves applications based on valid documentation, so the lack of a .edu email should not be a blocker.

Good luck!
